.tool-versions

```diff
@@ -0,0 +1,2 @@
+elixir 1.14.5-otp-25
+erlang 25.3.2
```

Comment:
would it be possible to bump this to the newer 1.16 for elixir and 26.2.1 for erlang? or is there a particular reason to keep these pinned at the older versions?